fluidkeys/main.go

```diff
@@ -267,6 +270,63 @@ func keyCreate() exitCode {
 	return 0
 }
 
+func keyList() exitCode {
+	fluidkeysDirectory, err := getFluidkeysDirectory()
+	if err != nil {
+		fmt.Printf("Failed to get fluidkeys directory")
+	}
+	db := database.New(fluidkeysDirectory)
+
+	keys, err := loadPgpKeys(db)
+	if err != nil {
+		panic(err)
+	}
+
+	emailAddressColumnWidth := len("Email address")
+
+	for _, key := range keys {
+		for _, id := range key.Identities {
+			emailAddressColumnWidth = Max(emailAddressColumnWidth, len(id.Name))
+		}
+	}
+
+	createdColumnWidth := 11
+	nextRotationColumnWidth := 13
+
+	header := fmt.Sprintf("%-*s  ", emailAddressColumnWidth, "Email address")
+	header += fmt.Sprintf("%-*s  ", createdColumnWidth, "Created")
+	header += fmt.Sprintf("%-*s  ", nextRotationColumnWidth, "Next rotation")
+
+	fmt.Printf("%s\n", colour.LightBlue(header))
+
+	printHorizontalRule(emailAddressColumnWidth, createdColumnWidth, nextRotationColumnWidth)
+
+	for _, key := range keys {
+		firstRow := true
```

Comment:
Some ascii-art documentation of how the table is supposed to look would be really helpful because I can't get in my head why this `firstRow` thing is necessary without thinking about the desired output